docs/source/projects.rst

```diff
@@ -87,6 +88,28 @@ Conda environment
   file in the root of the project directory or by including a ``conda_env`` entry in your
   ``MLproject`` file. For details, see the :ref:`project-directories` and :ref:`mlproject-specify-environment` sections.
 
+  The ``mlflow run`` command supports running a Conda environment project as a Python environment project.
+  To do this, run ``mlflow run`` with ``--env-manager local``:
+
+  .. code-block:: bash
+
+      mlflow run /path/to/conda/project --env-manager virtualenv
+
+  .. warning::
+
+      When a Conda environment project is executed as a Python environment project,
+      conda dependencies will be ignored and only pip dependencies will be installed.
+
+.. _project-python-environments:
+
+Python environment
```

Comment:
```suggestion
Virtualenv environment
```

Does `Virtualenv environment` sound strange because it repeats `env` twice?

Reply:
It sounds fine to me since it's the name of a tool :)